File: LABORATORIOS/LABORATORIO2/Ejercicio5/Java/Fibonacci.java
public class Fibonacci {
    public static int calcularSumaFibonacci(int n) {
        if (n <= 0) {
            return 0;
        }

        int suma = 0;
        int a = 0;
        int b = 1;

        for (int i = 0; i < n; i++) {
            suma += a;
            int temp = a;
            a = b;
            b = temp + b;
        }

        return suma;
    }

    public static void main(String[] args) {
        int nTerminos = 8;
        int sumaFibonacci = calcularSumaFibonacci(nTerminos);
        System.out.println(sumaFibonacci);
    }
}

File: LABORATORIOS/LABORATORIO2/Ejercicio5/Python/Fibonacci.py
def calcular_suma_fibonacci(n):
    if n <= 0:
        return 0

    suma = 0
    a, b = 0, 1

    for _ in range(n):
        suma += a
        a, b = b, a + b

    return suma

n_terminos = 8
suma_fibonacci = calcular_suma_fibonacci(n_terminos)
print(suma_fibonacci)
